pkgs/top-level/all-packages.nix

```diff
@@ -33271,4 +33273,5 @@ with pkgs;
   zthrottle = callPackage ../tools/misc/zthrottle { };
 
   zktree = callPackage ../applications/misc/zktree {};
+
```

Comment:
Please revert the addition of this line.

Reply:
Fixed on latest commits